src/application/controllers/Itinerary.php

```diff
@@ -138,7 +138,9 @@ public function generate_ics($departure, $arrival, $time_departure, $time_arriva
         $event = [
             'title' => "Voyage de Lausanne à Genève",
             'address' => "Voie 3" . $departure . $arrival,
-            'description' => "Détails de votre voyage du 14.04.19 : \\n\\n Départ : " . $time_departure . " de " . $departure . " sur Voie 2 \\n Arrivée : " . $time_arrival . "  à " . $arrival . " sur Voie 4\\n\\n Pour plus de détails : http://www.mtlaga.ch",
+            'description' => "Détails de votre voyage du 14.04.19 : \\n\\n Départ : " . $time_departure 
+                . " de " . $departure . " sur Voie 2 \\n Arrivée : " . $time_arrival . "  à " . $arrival 
+                . " sur Voie 4\\n\\n Pour plus de détails : " . base_url(),
             'datestart' => "2019-04-14 12:00:00",
             'dateend' => "2019-04-14 13:45:00"
```

Comment:
@hadrylouis hmmmmm,  hardcoded dates 😂. Imma open a new issue for date also

Reply:
euwwww brother whats that... Thanks for creating an issue. I'll work on that asap